### ISSUE TITLE
No current module handles site local service graph contract interfaces

### ISSUE THREAD

**Issue by manofcolombia** [CiscoDevNet/ansible-mso#275]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

* No current module handles site local service graph contract interfaces [CONTRACT_SITELOCAL_SERVICE_NODES_UPDATE]
* mso_schema_template_contract_service_graph takes care of binding a service graph template to a contract, but that is only half the configuration needed to be able to deploy a contract with a service graph attached.

### New or Affected Module(s):

* CONTRACT_SITELOCAL_SERVICE_NODES_UPDATE

### MSO version and MSO Platform

* 3.7(1j) all.

### APIC version and APIC Platform for Site Level Resources

* V 5.2.4e all.

### Collection versions

* cisco.mso latest nightly build from github actions

### References

![image](https://user-images.githubusercontent.com/2466902/175123850-e0ae8f3e-2b7c-4424-abb0-04b7ba39ebd4.png)

**Comment by phil-dotchon** [comment]
Seems to me this issue should not been closed as the PR has not been merged. It's a pity because all of the other service graph work already released is effectively useless until this final part is completed.

**Comment by akinross** [comment]
Hi @phil-dotchon, this issue is not closed yet. It is currently being worked on by @sajagana who has been involved with implementing this in terraform resources. There were issues on API side which caused delay and thus there was a hold period on the activity. Please follow this issue for updates.

**Comment by phil-dotchon** [comment]
> Hi @phil-dotchon, this issue is not closed yet. It is currently being worked on by @sajagana who has been involved with implementing this in terraform resources. There were issues on API side which caused delay and thus there was a hold period on the activity. Please follow this issue for updates.

Hi @akinross, OK sorry, my misunderstanding and poor attention, I should have spotted that the issue is still open!
I was a little frustrated as we, for a couple of years, have been using schema patching to implement service graphs and it's pretty painful; recently I've been looking for ways to improve speed and was excited to see the service graph modules. I started to migrate our code to use them but after some time I realised there was a gap/missing component and that it wasn't going to work, and then I discovered this issue.
